target_chains/ethereum/contracts/contracts/pulse/IPulse.sol

```diff
@@ -9,7 +9,7 @@ import "./PulseState.sol";
 interface IPulseConsumer {
     function pulseCallback(
         uint64 sequenceNumber,
-        address updater,
+        address provider,
```

Comment:
I'm wondering whether we really need to know the provider in the callback. the end user doesn't care, does it?

Reply:
yeah good point, will remove